flockoff/utils/git_utils.py

```
@@ -0,0 +1,140 @@
+    Returns:
+        True if the local commit is the latest, or if the check is disabled, or if remote URL cannot be determined.
+        False if the local commit is not the latest.
+        Prints warnings if git commands fail or if not in a git repository.
+    """
+    if os.environ.get(disable_check_env_var, 'false').lower() == 'true':
+        print(f"Commit check disabled via environment variable {disable_check_env_var}.")
+        return True
+
+    local_commit = get_local_commit_hash(local_repo_path)
+    if local_commit is None:
+        warnings.warn("Could not determine local commit hash. Skipping version check.")
```

Ruff (0.8.2)

No explicit `stacklevel` keyword argument found

(B028)